<!-- -->
<launch>
	<node pkg="data_collect" type="navsatfix_saver" name="navsatfix_saver_node" output="screen" clear_params="true">
		<param name="navsatfix_topic" type="string" value="/fix" />
		<param name="output_path" type="string" value="$(find data_collect)/output/fix/" />
	</node>
</launch>

